scripts/county_adjacency_msa/index.ts

```diff
@@ -65,6 +85,10 @@ function parseAdjacencyFile(filePath: string, delimiter = TAB): CountyItem[] {
   }));
 }
 
+function readDelimitedFile(text: string, delimiter = TAB) {
```

Comment:
Consider changing it so it receives the file path (so it actually reads the file instead of parsing the text)

```ts
function readDelimitedFile(filePath:string, delimiter=TAB) {
  // ...
}
```